js_modules/dagit/packages/ui/CHANGES.md

```diff
@@ -1,3 +1,7 @@
+## 1.0.8 (December 21, 2022)
+
+- Added ProductTour component
```

Comment:
Can you see if there are any other notable changes since 1.0.7?

Also, do we need to publish a new version already? Cloud uses master `dagit-core` and `ui`, so we don't need a new version there.

Reply:
https://github.com/dagster-io/dagster/commits/31fe69fb2817cf50aa39e65bcc1fec0ac3a91cc1/js_modules/dagit/packages/ui

I don't think there's any other notable change.

Reply:
> Also, do we need to publish a new version already? Cloud uses master dagit-core and ui, so we don't need a new version there.

We don't. We should start the log now though so that when we do publish its easier for whoever publishes later. I can remove the date and put (TBD) instead? I didn't actually update the version in package.json

Reply:
Ah, got it. Yeah, I saw the date next to the version number and assumed you were intending to publish.